### PR TITLE
fix: event invoice fetched correctly in review route

### LINKED ISSUE #3411: Event invoices are not fetched properly in /review route
**Describe the bug**

Currently, we use filter to get a particular event invoice. Also we are not redirecting request from review to paid even if it is paid

```
let filterOptions = [
      {
        name : 'identifier',
        op   : 'eq',
        val  : params.invoice_identifier
      }
    ];
```

**Expected behaviour**

We can use proper GET API to fetch invoice

**Additional context**

On it

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #3411

#### Short description of what this resolves:
Currently, we use filter to get a particular event invoice. Also we are not redirecting request from review to paid even if it is paid

#### Changes proposed in this pull request:
- fix event invoice query
- fix redirection of event invoice
- made minor changes to review page

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/routes/event-invoice/review.js [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-    return {
-      'user' : await this.authManager.currentUser,
-      'data' : await this.store.query('event-invoice', {
-        include : 'event',
-        filter  : filterOptions
-      })
-    };
+  titleToken(model) {
+    let invoice_identifier = model.get('identifier');
+    return this.l10n.t(`Review Event Invoice -${invoice_identifier}`);
```
Comment (CosmicCoder96) [inline app/routes/event-invoice/review.js]: tVar, and can't the variable be eliminated? it's being called just once.

Comment (shreyanshdwivedi) [reply app/routes/event-invoice/review.js]: @CosmicCoder96 eliminated the variable

File: app/routes/event-invoice/paid.js [hunk 2]
```diff
@@ -3,8 +3,7 @@ import Route from '@ember/routing/route';
 export default class extends Route {
 
   titleToken(model) {
-    let invoice_identifier = model.get('identifier');
-    return this.l10n.t(`Paid Event Invoice -${invoice_identifier}`);
+    return this.l10n.t(`Paid Event Invoice -${model.get('identifier')}`);
```
Comment (CosmicCoder96) [inline app/routes/event-invoice/paid.js]:
```js
return this.l10n.tVar(`Paid Event Invoice -${model.get('identifier')}`);
```

File: app/routes/event-invoice/review.js [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
-    ];
-    return {
-      'user' : await this.authManager.currentUser,
-      'data' : await this.store.query('event-invoice', {
-        include : 'event',
-        filter  : filterOptions
-      })
-    };
+  titleToken(model) {
+    return this.l10n.t(`Review Event Invoice -${model.get('identifier')}`);
```
Comment (CosmicCoder96) [inline app/routes/event-invoice/review.js]:
```js
return this.l10n.tVar(`Review Event Invoice -${model.get('identifier')}`);
```

Comment (shreyanshdwivedi) [reply app/routes/event-invoice/review.js]: Missed that :sweat_smile: 
Fixed it

**Pull request merged**